mappings/net/minecraft/world/gen/chunk/SlideConfig.mapping

```diff
@@ -12,6 +12,7 @@ CLASS net/minecraft/unmapped/C_rcuiqdtl net/minecraft/world/gen/chunk/SlideConfi
 	METHOD m_egwayqns applySlide (DD)D
 	METHOD m_fqfgwgiz (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
 		ARG 0 instance
+	METHOD m_pyjschec target ()D
```

Comment:
```suggestion
	METHOD m_pyjschec getTarget ()D
```

Reply:
Some of these suggestions are records, so I'll just do some quick checks before adding these suggestions